src/components/schema/jsonld.tsx

```diff
@@ -0,0 +1,92 @@
+import {LocalBusiness, Thing, WithContext, PostalAddress} from 'schema-dts';
+import PhotoGallery from '../photo-gallery';
+
+export function JsonLd<T extends Thing>(json: WithContext<T>): string {
+  return `<script type="application/ld+json">
+${JSON.stringify(json)}
+</script>`;
+}
+
+export const SchemaWrapper = (data: any) => {
+  return LocalBusinessSchemaWrapper(data)
```

Comment:
Seems like you only need the nested data anyways
```suggestion
  return LocalBusinessSchemaWrapper(data.document.streamOutput)
```